vnext/Microsoft.ReactNative/ReactApplicationDelegate.cpp

```diff
@@ -20,10 +20,27 @@ using namespace Windows::ApplicationModel::Activation;
 
 namespace winrt::Microsoft::ReactNative::implementation {
 
-static void ApplyArguments(ReactNative::ReactNativeHost const & /*host*/, std::wstring const &arguments) noexcept {
-  // Microsoft::ReactNative::implementation::ReactNativeHost* hostImpl {
-  // get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
+static void ApplyArguments(ReactNative::ReactNativeHost const &host, std::wstring const &arguments) noexcept {
+  Microsoft::ReactNative::implementation::ReactNativeHost *hostImpl{
+      get_self<Microsoft::ReactNative::implementation::ReactNativeHost>(host)};
   if (!arguments.empty() /*&& host.HasInstance()*/) {
+    constexpr wchar_t delimiter = L' ';
+    std::wistringstream argumentStream(arguments);
+    std::wstring token;
+    while (std::getline(argumentStream, token, delimiter)) {
+      if (token == L"-?") {
+        std::cout << "Options:" << std::endl
+                  << "  directDebugging <port>    Enable direct debugging on specified port." << std::endl;
```

Comment:
>directDebugging [](start = 24, length = 15)

I'd argue let's make these match the syntax of options elsewhere in the system --direct-debugging?

Reply:
Thanks for your input. Sounds good, changed all options to be consistent.